src/main/resources/application.yml

```
@@ -27,7 +27,7 @@ spring:
 
 
 
---- #prod
+--- # prod
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)